docs/CodeDevelopment/compile.md

```diff
@@ -63,8 +63,8 @@ the Pull Requests gates are implemented and enforced.
 
 ???+ info
     - In Project Mu repos the config file is generally at ```.pytool/CISettings.py```
-    - Project Mu runs on Windows 10 using the following tags: VS2019 and VS2022
-    - Project Mu runs on Ubuntu 18.04 using the tags: GCC5
+    - Project Mu runs on Windows 10/11 using the following tags: VS2019 and VS2022
```

Comment:
nit: Given [Windows 10 support ends on October 14, 2025](https://support.microsoft.com/windows/windows-10-support-ends-on-october-14-2025-2ca8b313-1946-43d3-b55c-2b95b107f281), we should officially move support only to Windows 11.